src/server/info_collector.cpp

```diff
@@ -46,28 +49,65 @@ info_collector::info_collector()
                                                                        "app_stat_interval_seconds",
                                                                        10, // default value 10s
                                                                        "app stat interval seconds");
+
+    _capacity_unit_stat_app = dsn_config_get_value_string(
+        "pegasus.collector", "capacity_unit_stat_app", "", "capacity unit stat app name");
+    dassert(_capacity_unit_stat_app.size() > 0, "");
```

Comment:
> 改为 `!_capacity_unit_stat_app.empty()`

这里和available_detector的写法是一样的，感觉两种写法没什么区别

Reply:
prefer empty() for readability
https://clang.llvm.org/extra/clang-tidy/checks/readability-container-size-empty.html
> The emptiness of a container should be checked using the empty() method instead of the size() method. It is not guaranteed that size() is a constant-time function, and it is generally more efficient and also shows clearer intent to use empty(). Furthermore some containers may implement the empty() method but not implement the size() method. Using empty() whenever possible makes it easier to switch to another container in the future.

Reply:
done